### PR TITLE
lab: add "coi" command

### PR DESCRIPTION
**Pull request by falzm**
This change introduces a new `exo lab coi` command, to deploy
Container-Optimized Instances.

### CODE REVIEW

**Comment by falzm** [comment]
[ch20397]

**Review by gabrielexoscale — APPROVED** [review]

**Review by illi-j — APPROVED** [review]
being able to specify environments when user doesn't choose the docker-compose option might be nice. What do you think?

otherwise LGTM! nice feature 👍

**Comment by falzm** [comment]
> being able to specify environments when user doesn't choose the docker-compose option might be nice. What do you think?

WDYM by *environment* ? 🤔

**Comment by illi-j** [comment]
> WDYM by _environment_ ?

docker run -e 

and the volumes too:
docker run -v

**Comment by falzm** [comment]
Ah, environment variables. Well IMHO this falls into the "advanced usage" category, which prompts a proper Docker Compose file based workflow in order to preserve the simplicity of the image based use case.

File: cmd/coi.go [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+		if err != nil {
+			return err
+		}
+
+		serviceOffering, err := getServiceOfferingByNameOrID(so)
+		if err != nil {
+			return err
+		}
+
+		template, err := getTemplateByNameOrID(zone.ID, coiTemplateName, "community")
```
Comment (pierre-emmanuelJ) [inline cmd/coi.go:114]: IMHO, You should use `Fedora CoreOS`, it's natively container optimized 😉

Comment (pierre-emmanuelJ) [reply cmd/coi.go:114]: Well, just checked, docker-compose is not installed by default on it 😕 only Docker

**Pull request merged**